.github/workflows/build_wamr_lldb.yml

```diff
@@ -111,7 +112,7 @@ jobs:
             -DLLVM_INCLUDE_TESTS:BOOL=OFF \
             -DLLVM_ENABLE_BINDINGS:BOOL=OFF \
             -DLLVM_ENABLE_LIBXML2:BOOL=ON \
-            -DLLDB_ENABLE_PYTHON:BOOL=OFF \
+            -DLLDB_ENABLE_PYTHON:BOOL=ON \
```

Comment:
Do we need to include a full version of python together with the LLDB download or what what is the common way of dealing with this dependency when shipping LLDB binaries?

Reply:
@wenyongh @lum1n0us 
Any ideas on how to create a distributable version of LLDB with python scripting support?